sandboxed-fs.js

```diff
@@ -4,11 +4,109 @@ const sandboxedFs = {};
 module.exports = sandboxedFs;
 
 const fs = require('fs');
+const pathModule = require('path');
+
+const errorMessage = 'path must be a string';
```

Comment:
That's strange, because I was checking it with the original `fs` module, and when I provide arguments of incorrect type to function, it returns errors starting from lowercase letters.
Here is an example:
![image](https://cloud.githubusercontent.com/assets/3501391/23089803/83052a54-f598-11e6-8a2b-cf2679dd21d5.png)